test/test_client.py

```diff
@@ -121,7 +121,7 @@ def test_read_multi_vars(self):
         byte_to_value = [util.get_real, util.get_real, util.get_int]
 
         # unpack and test the result of each read
-        for i in range(0, len(data_items)):
+        for i, _ in enumerate(data_items):
```

Comment:
sorry, but i don't think this code is an improvement

Reply:
It was claimed by Codefactor to use enumerate instead of range(len)) expression. So I changed it.